### PR TITLE
Signin and Signup functionality

### PR DESCRIPTION
**Pull request by shnawaz844**

### CODE REVIEW

File: src/apis/TicketAPI.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * restrictions set forth in your license agreement with F2 FINTECH.
+ */
+
+import { axiosInstance } from "@/apis/config/axiosConfig";
+import { defineCancelApiObject } from "@/apis/config/axiosUtils";
+
+export const TicketAPI = {
+  /** Login user
+   */
+  login: async (loginInfo, cancel = false) => {
```
Comment (coder-faraz) [inline src/apis/TicketAPI.jsx]: how will ticket login?

File: src/app/login/page.tsx [hunk 2]
```diff
@@ -25,27 +23,23 @@ import {
   Lock,
 } from "@mui/icons-material";
 import { ThemeProvider, useTheme, Theme } from "@mui/material/styles";
-import { url } from "inspector";
-
 import { UserAPI } from "@/apis/UserAPI";
+import { toast, ToastContainer } from "react-toastify";
```
Comment (coder-faraz) [inline src/app/login/page.tsx]: Toast component is already created, why using react-toastify

File: src/app/login/page.tsx [hunk 3]
```diff
@@ -153,121 +158,119 @@ const Login = (): JSX.Element => {
...
+                      color: "#fff",
+                      fontWeight: "bold",
+                      padding: "10px 20px",
+                      boxShadow: "0px 4px 15px rgba(0, 0, 0, 0.2)",
+                      transition: "all 0.3s ease",
+                      "&:hover": {
+                        background: "linear-gradient(45deg, #1976d2, #6A1B9A)",
+                      },
+                    }}
+                    disabled={isSubmitting}
```
Comment (coder-faraz) [inline src/app/login/page.tsx]: put dirty condition also in disabled property

File: src/app/user/UserForm.tsx [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+} from "@mui/icons-material";
+import { setLoading } from "@/redux/features/userSlice";
+import "react-toastify/dist/ReactToastify.css";
+import { UserAPI } from "@/apis/UserAPI";
+import { Formik, Form, Field } from "formik";
+import { toast, ToastContainer } from "react-toastify";
+import * as Yup from "yup";
+
+// Validation schema using Yup
+const UserSchema = Yup.object().shape({
```
Comment (coder-faraz) [inline src/app/user/UserForm.tsx:34]: use validation conditions as done in f2fintech website

**Review by coder-faraz — CHANGES_REQUESTED** [review]
comments left

File: src/apis/TicketAPI.jsx (hunk 1 above)
Comment (shnawaz844) [reply src/apis/TicketAPI.jsx]: done

File: src/app/login/page.tsx (hunk 2 above)
Comment (shnawaz844) [reply src/app/login/page.tsx]: done

File: src/app/login/page.tsx (hunk 3 above)
Comment (shnawaz844) [reply src/app/login/page.tsx]: done

File: src/app/user/UserForm.tsx (hunk 4 above)
Comment (shnawaz844) [reply src/app/user/UserForm.tsx:34]: done

**Review by shnawaz844 — COMMENTED** [review]
no conflicts

**Pull request merged**